process/factory/interceptorscontainer/shardInterceptorsContainerFactory_test.go

```diff
@@ -90,6 +91,10 @@ func createShardDataPools() dataRetriever.PoolsHolder {
 	pools.CurrBlockTxsCalled = func() dataRetriever.TransactionCacher {
 		return &mock.TxForCurrentBlockStub{}
 	}
+	pools.ProofsCalled = func() dataRetriever.ProofsPool {
+		return &dataRetrieverMock.ProofsPoolMock{}
```

Comment:
the mocked component should return by default this mocked proofs pool, so this would be avoided

Reply:
"called" fields are not set in this mocked component, so we have to set them here